pkg/cli/cmds/agent.go

```diff
@@ -167,11 +167,11 @@ var (
 	}
 	LBServerPort = cli.IntFlag{
 		Name:        "lb-server-port",
-		Usage:       "(agent/node) Internal Loadbalancer port",
+		Usage:       "(agent/node) Local port for supervisor tunnel load-balancer. If the supervisor and apiserver are not colocated; an additional port 1 below this one will also be used for the apiserver tunnel.",
```

Comment:
I can't help but feel that the description here is a little bit vague, I can understand it as `1 below this one` i.e. 1 below 6444 is 6443 but I cannot say the same for everyone.

Reply:
Just tweaked it a bit, is that any better?

Reply:
Yeah, that looks great! Thanks